### PR TITLE
kakoune plugins and update.py update

### PR DESCRIPTION
**Pull request by dgbuckley**
###### Description of changes

Update kakoune plugins and updated the kakoune update.py script so it runs

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/editors/kakoune/plugins/generated.nix [hunk 1]
```diff
@@ -1,235 +1,212 @@
...
-      sha256 = "13jdyd2j45wvgqvxdzw9zww14ly93bqjb6700zzxj7mkbiff6wsb";
-    };
-    meta.homepage = "https://github.com/occivink/kakoune-vertical-selection/";
-  };
-
-  openscad-kak = buildKakounePluginFrom2Nix {
-    pname = "openscad-kak";
-    version = "2020-12-10";
-    src = fetchFromGitHub {
+    }
```
Comment (teto) [inline pkgs/applications/editors/kakoune/plugins/generated.nix:170]: it lacks a `;`

**Comment by teto** [comment]
sry I forgot to update the kakoune updater (first time so far). I've checked out the ranch and ran the kakoune update script but it choked on missing `;`;  line 15

File: pkgs/applications/editors/kakoune/plugins/generated.nix (hunk 1 above)
Comment (dgbuckley) [reply pkgs/applications/editors/kakoune/plugins/generated.nix:170]: Good catch, I'll update. Thanks!

**Comment by mirrexagon** [comment]
I have a pull request also trying to do the same thing here: https://github.com/NixOS/nixpkgs/pull/169140

**Comment by AndersonTorres** [comment]
Closing as dead and conflicting.

**Pull request closed without merging**